src/osgPlugins/tga/ReaderWriterTGA.cpp

```diff
@@ -867,7 +872,7 @@ class ReaderWriterTGA : public osgDB::ReaderWriter
 
             osgDB::ifstream istream(fileName.c_str(), std::ios::in | std::ios::binary);
             if(!istream) return ReadResult::FILE_NOT_HANDLED;
-            ReadResult rr = readTGAStream(istream);
+            ReadResult rr = readTGAStream(istream, options && options->getOptionString().find("IGNORE_TGA_2_FIELDS") != std::string::npos);
```

Comment:
Options are typically not DONE_LIKE_MACROS in OSG. Rather, they use camelCase or, rarely, snake_case.

Reply:
I copied the JPEG one https://github.com/openscenegraph/OpenSceneGraph/blob/OpenSceneGraph-3.6/src/osgPlugins/jpeg/ReaderWriterJPEG.cpp#L811

Are you sure that's the outlier?